Zahlenraten.py

```diff
@@ -0,0 +1,84 @@
+import random
+
+game = True
+guess = 0
+distance = 0
+
+print("Hallo, wie heißt du?")
+name = input()
+
+while game == True:
+
+    num = random.randint(1, 100)
+    attempts = 0
+
+    print("Hallo " + name + "! Ich denke an eine Zahl zwischen 1 und 100. Welche ist es?")
+
+    while True:
+
+        tempNumber = input()
```

Comment:
```suggestion
        currentGuess = input()
```

You don't know if this a number yet